requirements-devel

```diff
@@ -0,0 +1,7 @@
+
+-r requirements.txt
+-r requirements-docs.txt
+-r requirements-tests.txt
+
+jupyter
```

Comment:
In theory, `jupyter` is not required to build the documentation as sphinx does all the needful. I did not want to use sphinx in our doc and integration tests though so I install `jupyter` separately. I do not have strong feelings so if you prefer to add it to `requirements-docs.txt`, it would be fine by me.

Reply:
I'll add it then.  In my interpretation `requirements-docs.txt` also installs the tools which one needs to work on the documentation - which now included `jupyter`...

Reply:
done.